app.js

```diff
@@ -237,24 +326,22 @@ async function watchRandomStreamers(browser, page) {
     }
 
     // Is this streamer still streaming?
-    const channelStatus = await page
-      .$eval(CHANNEL_STATUS_QUERY, (el) => el.textContent)
-      .catch(() => "");
+    const channelStatus = (await query(page, CHANNEL_STATUS_QUERY)).text().trim().toUpperCase();
     info("Channel status: " + channelStatus);
     // We use `startsWith` because sometimes we get LIVELIVE
     // Also `toUpperCase` because sometimes we get LiveLIVE
     // This is because there are two elements with that class name
     // One below the player and one "in" the player
-    if (!channelStatus.toUpperCase().startsWith("LIVE")) {
+    if (!channelStatus.includes("LIVE")) {
```

Comment:
We still need the `toUpperCase` due to the comment that I have above

Reply:
Check the reply above

Reply:
Ah I didn't see you'd added `toUpperCase` directly on the declaration, nice